package.json

```
@@ -113,6 +114,7 @@
     "mailparser": "3.6.4",
     "markdown-it": "13.0.1",
     "module-alias": "2.2.2",
+    "notion-to-md": "^3.0.1",
```

```suggestion
    "notion-to-md": "3.0.1",
```
The exact semver match is preferred. `pnpm-locked.yaml` needs to be updated as well.